packages/core/src/components/ozon-content/nodes/figuur.node.tsx

```diff
@@ -4,6 +4,8 @@ import { getNodeName } from "../get-node-name.function";
 import { OzonContentNodeContext } from "../ozon-content-node-context.interface";
 import { OzonContentNode } from "../ozon-content-node.interface";
 
+export type FiguurEditAction = "voegtoe" | "verwijder";
```

Comment:
Alle public facing types/interfaces namespacen we met de naam van het component. Dit zou dus zijn `OzonContentFiguurEditAction`.

Reply:
Wat ik mij nog afvraag: Je hebt hier en daar een vertaling doorgevoerd (voegtoe -> add, verwijder -> remove).

Is het nog duidelijk dat de Nederlandse taal gaat over de bron (STOP) en dat de Engelse vertaling over ons domein gaat?

Want ik zou hier een van de twee verwachten:

```
export type FiguurEditAction = "add" | "remove";
export type FiguurWijzigactie = "add" | "remove";
```

De mix volg ik nog niet helemaal.